src/api/pagination.md

```diff
@@ -11,6 +11,12 @@ permalink: /api/pagination
 Pagination allows a listing endpoint to return a subset of results. The goal is to reduce memory
 usage and speed up page rendering.
 
+To retrieve the next page, you can expect to be able to supply `nextPageKey` with a `pageKey` query
+parameter to get at the next page of results.
```

Comment:
To retrieve the next page, a `pageKey` can be supplied. This will be the `nextPageKey` returned from your previous query.

Reply:
I agree with this.